.github/workflows/pr-graphql-compat-check.yml

```diff
@@ -23,7 +21,8 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        release: ['15.5.3', '^15.8.0', '16.1.0', '16.2.0', '16.3.0']
+        # test all peer ranges
+        release: ['^15.5.0', '^15.8.0', 'latest']
```

Comment:
should we really test 15.5.0? or only 15.8.0

Reply:
@B2o5T I'm testing the lowest of the peer ranges, in `codemirror-graphql` we set that as the lowest